crates/sui-indexer-alt/src/models/checkpoints.rs

```diff
@@ -13,3 +14,13 @@ pub struct StoredCheckpoint {
     /// BCS serialized CheckpointContents
     pub checkpoint_contents: Vec<u8>,
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_stored_checkpoint_field_count() {
+        assert_eq!(StoredCheckpoint::field_count(), 3);
```

Comment:
@gegaowp, can you remove the tests? They defeat the point of automatically calculating the field count because you will always need to update the tests...

Reply:
by having the automatic calculation we can ensure the value is right, vs checking manually -- but I got your point too the test should be about FieldCount itself so better to test it there.